test/packages/apache/data_stream/error/_dev/test/system/test-default-config.yml

```diff
@@ -1,3 +1,6 @@
+skip:
+  reason: testing skip
+  link: https://github.com/elastic/integrations/issues/123456789
 vars: ~
 data_stream:
   vars:
```

Comment:
I assume that this test will fail if it runs, that's why it's skipped = PASS

Reply:
This used to be a test that used to pass before I added the skip. Are you suggesting that I should make it fail, so we have a way to know something is not working correctly if the skip logic in the test runner breaks?

Reply:
Yes, that's point :) I think the quick way is to introduce some faulty variables? Don't have to implement anything special. But I wouldn't say it's blocking comment. For me this PR is ready to be merged.

Reply:
Thanks for confirming, I will make this change now. Should be fairly quick to test.

Reply:
I'm going to do a series of 3 commits to test this change:
1. A commit that comments out the `skip` section, effectively re-enabling the `apache.error` system test. With this commit the test should pass.
2. A commit that breaks the test. With this commit the test should fail.
3. A commit that adds back the `skip` section. With this commit the test should be skipped.